src/Validation/DonationValidator.php

```diff
@@ -49,6 +64,12 @@ public function validate( $donation ): bool {
 		return empty( $this->constraintViolations );
 	}
 
+	public function needsModeration( Donation $donation ): bool {
+		// TODO: add TextPolicyValidator
+		$this->policyViolations[] = $this->amountPolicyValidator->validate( $donation->getAmount() );
```

Comment:
I'm thinking that the interval parameter should probably be required. However this call does not have access to that info, as it is not in Donation. Any plan on how to solve that yet?